proto-lens-discrimination/src/Data/ProtoLens/Discrimination.hs

```diff
@@ -0,0 +1,144 @@
+-- | This module implements discriminators of protobufs generically.
+
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Data.ProtoLens.Discrimination
+    ( discDescriptor
+    , discField
+    , discText
+    , discByteString
+
```

Comment:
Remove blank line.  Or alternately, add a Haddock header to distinguish; but since this module is so small I don't know if it's needed.